### DISCUSSION TITLE
Support a mask window?

### DISCUSSION THREAD

**Discussion by evbo** [elrumordelaluz/reactour#474] [Ideas]
I notice in the examples that the mask completely covers the screen. Is there a way for the mask to cover everything _except_ the DOM element(s) you are trying to provide a tour of?

For instance, like is done in Atlaskit? I am trying to migrate away from Atlaskit but this is the one feature that's hard to live without:
https://atlassian.design/components/onboarding/examples

**Comment by elrumordelaluz** [comment]
Hi  @evbo, the _Mask_ without extra config covers all the screen except the highlighted element. Is this what you are looking for?

**Reply by evbo** [reply]
Thanks! Yes, so I guess my browser (Brave, Ubuntu 20.04) wasn't displaying that correctly in the official demos. Odd.
Version 1.38.119 Chromium: 101.0.4951.67 (Official Build) (64-bit)

I also use Dark Reader, so maybe that was it...

**Reply by elrumordelaluz** [reply]
can you please share a screenshot and a the info to reproduce it? thanks!

**Reply by evbo** [reply]
sure! repro info was given above (browser / os version). Technically I'm using "Xubuntu" 20.04 though, which is a flavor of Ubuntu

Here's the screen with everything masked including the target:
![delete](https://user-images.githubusercontent.com/6313004/170731220-2e177671-0399-4a33-ac69-e71229a8f319.png)

From the production website: https://reactour.js.org/

**Reply by elrumordelaluz** [reply]
wow, pretty weird! will try to reproduce in order to find possibles fixes. Thanks for pointing out!

The examples [here](https://reactour.vercel.app/) have the same result?
In case you have any input, regarding a solution, any PR is so welcome!
Thanks!

**Reply by evbo** [reply]
And those examples seem to work better, yes!

**Reply by elrumordelaluz** [reply]
ok, thank you!
